extensions/amp-apester-media/0.1/test/test-amp-apester-media-utils.js

```diff
@@ -26,6 +27,9 @@ describes.realWin('amp-apester-media-utils', {}, (unused) => {
     document.body.textContent = '';
     document.head.textContent = '';
   });
+  afterEach(() => {
+    Services.ampdoc(document).meta_ = null;
```

Comment:
Instead of accessing the private property, let's reset using `setMetaByName` -- `keywords` is the only one that needs to be reset for the purposes of tests, right?

Reply:
I suppose we could do that, but this is the most direct means of resetting the cached meta tags in an AmpDoc, see: https://github.com/ampproject/amphtml/blob/master/testing/describes.js#L739-L740 . My preference would be to leave this as-is, but let me know if it needs to change.

Reply:
OK, feel free to leave as is. If you could include the same comment that would be helpful too.